core/featureflags/src/main/kotlin/app/k9mail/core/featureflag/FeatureFlagResult.kt

```diff
@@ -5,6 +5,14 @@ sealed interface FeatureFlagResult {
     data object Disabled : FeatureFlagResult
     data object Unavailable : FeatureFlagResult
 
+    fun <T> get(
```

Comment:
It think `get` is not the right function name. What about `fold` from functional programming, which could be used when reducing or combining data. Or `map`, but it's more related to lists. A long version `whenEnabledOrNot`. What do you think?